source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -82,7 +89,7 @@ see the :ref:`node-fundamentals-query-document` guide.
    You can specify options in a find operation even when you pass an
    empty query. For example, the following code shows how you can
    specify a projection as an option while executing a find operation
```

Comment:
You can link to subsections! I ended up linking to the projection guide in the Node docs instead, since it seemed a little more useful than the manual page.